react-native-cli/index.js

```diff
@@ -136,12 +138,12 @@ function createProject(name) {
   fs.writeFileSync(path.join(root, 'package.json'), JSON.stringify(packageJson));
   process.chdir(root);
 
-  console.log('Installing react-native package from npm...');
-  exec('npm install --save react-native', function(e, stdout, stderr) {
+  console.log('Installing ' + REACT_NATIVE_PACKAGE_ID + ' package from npm...');
+  exec('npm install --save ' + REACT_NATIVE_PACKAGE_ID, function(e, stdout, stderr) {
```

Comment:
Ah, yeah. Forgot about this one since it was meant for the internal use and as it's usually run by the user on its machine.